lib/rubocop/cop/rspec/each.rb

```diff
@@ -0,0 +1,53 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that all matcher is used instead of iterating over an array.
+      #
+      # @example
+      #   # bad
+      #   it 'validates users' do
+      #     [user1, user2, user3].each { |user| expect(user).to be_valid }
+      #   end
+      #
+      #   # good
+      #   it 'validates users' do
+      #     expect([user1, user2, user3]).to all(be_valid)
+      #   end
+      class Each < Cop
```

Comment:
I agree that `Each` is too generic a name. `IteratedExpectation` is the best I can think of right now, and I don’t really like that either. Perhaps @backus can think of a better name?